lpc55_sign/src/signed_image.rs

```diff
@@ -13,18 +13,40 @@ use rsa::{
     pkcs1::DecodeRsaPrivateKey, pkcs1::DecodeRsaPublicKey, pkcs8::DecodePrivateKey, PublicKeyParts,
     RsaPrivateKey, RsaPublicKey,
 };
+use serde::Deserialize;
 use sha2::{Digest, Sha256};
 use x509_parser::parse_x509_certificate;
 
-#[derive(Clone, Debug, Parser)]
+#[derive(Clone, Debug, Deserialize)]
+#[serde(rename_all = "kebab-case", deny_unknown_fields)]
+pub struct CertConfig {
```

Comment:
I think this struct is only of use to the binary, so I might move it to `lpc55_sign_bin/src/main.rs`. By the time certs get into this library code, they should already be in memory.

Reply:
This was _originally_ in the binary, but I realized that it was duplicating some code in Hubris, so I consolidated it here:
https://github.com/oxidecomputer/hubris/blob/hubtools-signing/build/xtask/src/config.rs#L567

Reply:
I see. That makes sense, thanks.